### PR TITLE
Fix npm

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR fixes NPM conflict.

### CODE REVIEW

File: contracts/binding/src/lib.rs [hunk 1]
```diff
@@ -2,31 +2,25 @@
 #[macro_use]
```
Comment (fridrik01) [inline contracts/binding/src/lib.rs:2]: is this change needed? I thought we forced this to be formatted in https://github.com/consensus-shipyard/ipc/pull/659

**Comment by dnkolegov** [comment]
> is this change needed? I thought we forced this to be formatted in https://github.com/consensus-shipyard/ipc/pull/659

@fridrik01 
This is generated during CI; git on my machine shows no changes in that file.
